src/Router.tsx

```diff
@@ -41,8 +45,15 @@ const Router: React.FC = () => {
         <Route exact path={PATHS.TUTORIAL_PAGE}>
           <TutorialPage />
         </Route>
+        <AuthRoute path={PATHS.STUDY_VIEW_PAGE}>
+          <StudyViewPage />
+        </AuthRoute>
+        {/* Not sure why this needs to come after the view route, but it only works that way */}
```

Comment:
I reproduced, locally, the behavior you alluded to in the code comment. In other words, when the "create" route is above the "view" route, the "view" route is the one that the app follows and the app shows the string `create` as though it were the ID of a study.

<img src="https://github.com/microbiomedata/nmdc-field-notes/assets/134325062/7bb9f15f-33c1-4561-94b5-de83716d1176" width="400" />

It doesn't make sense to me. Your code comment matches my observations. I also tried replacing the `AuthRoute`s with regular `Route`s and that didn't seem to me to make any difference.

Reply:
While playing around with this phenomenon locally; in an attempt to eliminate some variables, I came up with this example that doesn't use any custom components:

![image](https://github.com/microbiomedata/nmdc-field-notes/assets/134325062/3a381668-9821-4795-9dc4-4984a2aefad7)

Reply:
I found that, if I wrap the contents of the `<IonRouterOutlet>` element in a `<Switch>` element (where `Switch` is imported from `react-router-dom`), the routing **does behave the way I expect**.

```diff
- import {Redirect, Route} from "react-router-dom";
+ import {Redirect, Route, Switch} from "react-router-dom";

// ...

    <IonReactRouter>
      <IonRouterOutlet>
+       <Switch>

// ...

+       </Switch>
      </IonRouterOutlet>
    </IonReactRouter>
```

The Ionic [documentation](https://github.com/ionic-team/ionic-docs/blob/4df75fbf279f1e4901bad988a865a39a9d738c1b/docs/react/navigation.md?plain=1#L583-L585) says:

> Since `IonRouterOutlet` takes over the job in determining which routes get rendered, using a `Switch` from React Router has no effect when used inside of an `IonRouterOutlet`.

That is not consistent with what I am seeing (assuming, where it says "from React Router," they are not making a distinction between `react-router` and `react-router-dom`).

Reply:
I'm willing to proceed with it for now, too. I like that you pointed it out in the code comment. 

I will create an issue about it (not assigned to any person or sprint) for future reference.